examples/full-example/.mendelrc

```diff
@@ -53,23 +53,19 @@ transforms: # a list of all available transforms for all envs and types
         - istanbul
 
 types:
-  json:
+  json: # Type id. Doesn't have to be file extension.
     extensions:
       - .json
-    parser: json-parse
-    parser-to-type: js
-    include-node_modules: true
+    parser: json-parse # Uses parser defined in transforms.
+    parser-to-type: js # Maps to type defined below
```

Comment:
AFAIK `parser:` is optional, and `parser-to-type:` is likely mandatory only when `parser:` is present. Comments should say that.